pkg/operator/ceph/object/spec.go

```diff
@@ -319,8 +319,15 @@ func (c *clusterConfig) reconcileService(cephObjectStore *cephv1.CephObjectStore
 	if err != nil {
 		return "", errors.Wrapf(err, "failed to create or update object store %q service", cephObjectStore.Name)
 	}
+	var port int32
+
+	if cephObjectStore.Spec.Gateway.SecurePort != 0 && cephObjectStore.Spec.Gateway.SSLCertificateRef != "" {
+		port = cephObjectStore.Spec.Gateway.SecurePort
+	} else if cephObjectStore.Spec.Gateway.Port != 0 {
+		port = cephObjectStore.Spec.Gateway.Port
+	}
+	logger.Infof("ceph object store gateway service running at %s:%d", svc.Spec.ClusterIP, port)
```

Comment:
If they specified both the port and securePort, it will be listening on both ports. How about we just don't log the port, and you could delete lines 322-328?
```suggestion
	logger.Infof("ceph object store gateway service running at %s", svc.Spec.ClusterIP)
```

Reply:
Done.